tests/Foundation/Http/Middleware/TrimStringsTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Illuminate\Tests\Foundation\Http\Middleware;
+
+use Illuminate\Foundation\Http\Middleware\TrimStrings;
+use Illuminate\Http\Request;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpFoundation\Request as SymfonyRequest;
+
+class TrimStringsTest extends TestCase
+{
+    public function testTrimStringsIgnoringExceptAttribute() : void
+    {
+        $middleware = new TrimStringsWithExceptAttribute();
+        $symfonyRequest = new SymfonyRequest([
+            'abc' => '  123  ',
+            'xyz' => '  456  ',
+            'foo' => '  789  ',
+            'bar' => '  010  ',
+        ]);
+        $symfonyRequest->server->set('REQUEST_METHOD', 'GET');
+        $request = Request::createFromBase($symfonyRequest);
+
+        $middleware->handle($request, function (Request $request) {
+            $this->assertEquals('123', $request->get('abc'));
```

Comment:
Maybe it would be better to use `assertSame` method?

Reply:
@antonkomarev maybe.. in this case what do you think about replacing `assertEquals` with `assertSame` in `TransformsRequestTest` class test as well?

https://github.com/laravel/framework/blob/4fb409cd703ce79fad259a406e8ef59885f32568/tests/Foundation/Http/Middleware/TransformsRequestTest.php#L24